src/fhir/guides/ig-atf/Home/Einfuehrung.page.md

```diff
@@ -1,41 +1,76 @@
+Mit dieser Spezifikation wird der strukturierte Datenaustausch auf Basis von FHIR weiterentwickelt und ermöglicht die flexible Nutzung über verschiedene Austauschplattformen und -protokolle wie KIM und TIM. Ziel ist es, eine einheitliche, interoperable Lösung für den sicheren, effizienten Datenaustausch zu schaffen.
 
-## Konzept
+## Wie funktioniert das ATF?
 
-Diese Spezifikation basiert auf dem "Messaging"-Konzept von FHIR. Dafür gibt es die Profile `BundleAppTransportFramework` (MessageBundle) und `MessageHeaderAppTransportFramework` (MessageBundle).
+Das App-Transport-Framework (ATF) ermöglicht eine strukturierte Kommunikation zwischen einem Sender und einem Empfänger, basierend auf standardisierten FHIR-Ressourcen. Die gesendete Nachricht enthält einen Betreff, der den Nachrichtentyp im jeweiligen Anwendungsfall festlegt, und einen Nachrichtentext, der bei Bedarf über XSLT generiert werden kann. Ein FHIR Bundle wird als Anhang hinzugefügt, das die eigentlichen Daten der Nachricht enthält.
 
-Das Bundle stellt hierbei den Container für den Nachrichtenaustausch bereit und referenziert in .entry auf einen MessageHeader. Dieser MessageHeader hält die Informationen von Sender und Empfänger, analog zur Kommunikation via E-Mail, sowie unter .eventCoding einen Code der angibt, zu welchem Anwendungsfall die gesendete Nachricht zuzuordnen ist.
+Im FHIR Bundle befindet sich ein **MessageHeader**, der wichtige Informationen wie die Absenderadresse, das sendende System und den Code des Anwendungsfalls enthält. Dieser Code gibt an, um welche Art von Nachricht es sich handelt (z.B. Rezeptanforderung oder Befund). Die konkrete medizinische Information, die übertragen wird, ist in den **FHIR-Ressourcen des Anwendungsfalls** enthalten. Sollte eine maschinelle Verarbeitung nicht möglich sein, kann optional ein menschenlesbares PDF erstellt werden, das aus den FHIR-Ressourcen mithilfe eines XSLT-Stylesheets generiert wird.
 
-Derzeit sind KIM und TIM als Austauschmedien angedacht, was sich in Zukunft noch erweitern lässt.
+Nach dem Senden der Nachricht erhält der Absender eine automatisierte **Empfangsbestätigung** vom Empfänger. Der Anhang der Antwort umfasst ein weiteres FHIR Bundle mit einem MessageHeader, der neben den üblichen Absender- und Empfängerinformationen auch die ID der ursprünglichen Nachricht enthält, die bestätigt wird.
 
-Das Feld `.focus` im MessageHeader verweist dann auf eine oder mehrere Ressourcen, die im Fokus des jeweiligen Anwendungsfalls stehen. Dies variiert und wird dann von den Projekten festgelegt, die diese Spezifikation umsetzen.
+Zusätzlich enthält die Antwort das Ergebnis der Nachrichtenauswertung. Dies gibt an, ob die Nachricht erfolgreich verarbeitet wurde oder ob ein Fehler aufgetreten ist. Bei einem Fehler wird eine detaillierte Fehlermeldung zurückgesendet, damit das Problem identifiziert und behoben werden kann.
+
+![ATF Message Flow](src/fhir/guides/ig-atf/images/ATF-Message-Flow.png)
```

Comment:
Link funktioniert nicht.
Am besten mit {{pagelink:}} vom simplifier ig arbeiten

Reply:
geändert in 
> {{render:guides/ig-atf/images/ATF-Message-Flow.png}}

Reply:
ich nehme an, dass das richtig ist :D